danceradvent/public/articles/2023/21-dancer-core-team-partnership-with-tprf.pod

```diff
@@ -0,0 +1,70 @@
+=head1 Tidings of Comfort and Joy: Dancer Core Team, TPRF Partnership
+
+In the lead up to Dancer2 1.0.0, the Dancer Core Team finalized an
+arrangement with The Perl and Raku Foundation (TPRF) to have the
+foundation collect and manage funds on behalf of the Dancer project.
+
+=head2 Background
+
+We're a small team, and setting up and managing a non-profit corporation
+is a significant time investment. Every minute we spend managing funds,
+paying bills, etc. is time we aren't spending advancing Dancer. TPRF
+has all of this infrastructure in place already, and such a partnership
+helps TPRF fulfill its goal of supporting the Perl and Raku communities.
+
+From the perspective of the Dancer Core Team, this is a huge win; TPRF
```

Comment:
There's a jump here to "huge win" without enough explanations on what this partnership is about and how it works.

I think this paragraph ("From the perspective of the...") should be further below after explaining the partnership.